models/modelArticles.js

```diff
@@ -48,4 +50,49 @@ function retrieveAllArticleData(sort_by = "created_at", order = "DESC") {
   });
 }
 
-module.exports = { retrieveArticleDataById, retrieveAllArticleData };
+function retrieveArticleComments(
```

Comment:
None of this sortby or order by is required as it is not going to be passed by the client. We just need to make the SQL query shape our data as default. The only thing that should be dynamic is the article id